subprojects/wrapper/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -71,18 +89,28 @@ public File call() throws Exception {
 
                 verifyDownloadChecksum(configuration.getDistribution().toString(), localZipFile, distributionSha256Sum);
 
-                logger.log("Unzipping " + localZipFile.getAbsolutePath() + " to " + distDir.getAbsolutePath());
                 unzip(localZipFile, distDir);
 
                 File root = getAndVerifyDistributionRoot(distDir, safeDistributionUrl.toString());
                 setExecutablePermissions(root);
                 markerFile.createNewFile();
 
+                printReleaseSummary();
+
                 return root;
             }
         });
     }
 
+    private void printReleaseSummary() {
+        if (gradleVersion != null) { // appVersion could be null in custom distributions
+            logger.log("");
+            logger.log("Welcome to Gradle " + gradleVersion + "!");
+            logger.log("See what's new at https://gradle.org/releases#" + gradleVersion);
```

Comment:
:o: Would be awesome if we could make the URL clickable by default from the console (similar to what we do for failed test results).